configs/vsphere-config-win.yml.sample

```diff
@@ -1,55 +1,58 @@
 integrations:
   - name: com.newrelic.vsphere
     exec:
-    # Do not change this path
       - C:\Program Files\New Relic\newrelic-infra\newrelic-integrations\bin\nri-vsphere.exe
-      
+    env:
```

Comment:
Do the docs reflect this change?
If I remember correctly there is a direct link to the github page, anything else required?

Reply:
By the way, it is really better with this format

Reply:
not yet, i'm updating that also. thanks for the reminder